flake.nix

```diff
@@ -0,0 +1,44 @@
+{
+  description = "Dev environment";
+
+  outputs = { self, nixpkgs }:
+    let pkgs = nixpkgs.legacyPackages.x86_64-linux;
+    in {
+      defaultPackage.x86_64-linux =
+        pkgs.mkShell {
+          buildInputs =  with pkgs; [
+            (php82.withExtensions
+              ({ all, ... }: with all; [
```

Comment:
Not sure if @j0k3r wants to include Nix packaging. I could continue to maintain it for the time being as I use something like this locally.

----

It might be sufficient to use `enabled` attribute to load the default extensions:

https://github.com/fossar/selfoss/blob/b0eb43d80bedde5da3204b842511afe245d8c17c/flake.nix#L37-L40

And make the PHP version parametric:

https://github.com/fossar/guzzle-transcoder/blob/816585c89c87268d1188d2d9696b49ce73541ff9/flake.nix#L29

Reply:
If you can maintain it, I'm ok.
I didn't what Flake was.
Maybe put a comment at the very top of `flake.nix` with a link to sth that explain what it does/is.